<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADFG(V)X</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButtonADFGX">
      <property name="text">
       <string>ADFGX</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButtonCzech">
      <property name="text">
       <string>Česká tabulka</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="radioButtonADFGVX">
      <property name="text">
       <string>ADFGVX</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="radioButtonEnglish">
      <property name="text">
       <string>Anglická tabulka</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="chybejici_znaky">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="klica">
      <property name="text">
       <string>Klíčové slovo</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditKlicoveSlovo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="opentext">
      <property name="text">
       <string>Zadejte text pro zašifrování</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditTextSifrovani"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="sifrovanytext">
      <property name="text">
       <string>Zadejte text pro dešifrování</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEditTextDesifrovani"/>
    </item>
    <item row="8" column="0" rowspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Zadejte tabulku:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEditTextTabulka"/>
    </item>
    <item row="9" column="1">
     <widget class="QRadioButton" name="radioButtonNahodna">
      <property name="text">
       <string>Náhodná tabulka</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QRadioButton" name="radioButtonRucni">
      <property name="text">
       <string>Ručně zadaná tabulka</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="chybiGUI">
      <property name="text">
       <string>Chybějící znaky:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="zasifrovanyTextGUI">
      <property name="text">
       <string>Zašifrovaný text</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="desifrovanyTextGUI">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Dešifrovaný text</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="vyfiltrovanyTextGUI">
      <property name="text">
       <string>Vyfiltrovaný text:</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="tabulkaGUI">
      <property name="text">
       <string>Tabulka pro šifrování:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QPushButton" name="pushZasifrovat">
      <property name="text">
       <string>Zašifrovat</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QPushButton" name="pushDesifrovat">
      <property name="text">
       <string>Dešifrovat</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
